adhocteam/Head-Start-TTADP
# Allow search to be case-insensitive

**Description of change**
User search is now case-insensitive in the admin page

**How to test**
1. Pull down changes
2. Go to admin page
3. Type a users name in a mixture of upper/lower and you should still get the desired user (if exists)

**Issue(s)**
* https://github.com/HHS/Head-Start-TTADP/issues/274

**Checklist**

    - API methods
    - Boundary and Data Flow Diagrams
    - [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions with the [Nygard template](https://github.com/joelparkerhenderson/architecture_decision_record/blob/master/adr_template_by_michael_nygard.md)
    - OSCAL templates completed when security controls are implemented or modified
